.eslintrc.json

```diff
@@ -0,0 +1,22 @@
+{
+    "env": {
+        "browser": true,
+        "es6": true
+    },
+    "extends": [
+        "airbnb-base"
+    ],
+    "globals": {
+        "Atomics": "readonly",
+        "SharedArrayBuffer": "readonly"
+    },
+    "parserOptions": {
+        "ecmaVersion": 2018,
+        "sourceType": "module"
+    },
+    "rules": {},
+    "ignorePatterns": [
+        "dist/app.bundle.js",
+        "node_modules/"
+    ]
+}
```

Comment:
Please do not change the linter file ![see this](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/linter_config.png)

Reply:
Ohh absolutely right! I forgot to add the recommended microverse config, thank you for noticing!

I am also keeping the ignore patterns, as they are just the result of using webpack and npm, not code written by myself.